libr/include/r_util/r_str.h

```diff
@@ -23,12 +24,39 @@ typedef int (*RStrRangeCallback) (void *, int);
 
 static inline void r_str_rmch(char *s, char ch) {
 	for (;*s; s++) {
-		if (*s==ch) {
+		if (*s == ch) {
 			memmove (s, s + 1, strlen (s));
 		}
 	}
 }
 
+//  -- strf
+typedef struct r_strf_t {
+	size_t idx;
+	size_t x;
+	size_t y;
+	char *buf[0];
+} r_strf__;
+
+static inline char *r_strf_(struct r_strf_t *s, const char *fmt, ...) {
+	if (s->idx >= (s->y * s->x)) {
+		s->idx = 0;
+	}
+	r_return_val_if_fail (s && fmt, NULL);
```

Comment:
`s` is already used above, it's useless to check it here.

Reply:
@radare this comment was done 7 days ago and it is still not resolved AFAIS

Reply:
solved, sorry i dont use the web all the time